fbw-a32nx/src/systems/fmgc/src/components/index.ts

```diff
@@ -27,5 +30,5 @@ export function updateComponents(deltaTime: number): void {
 }
 
 export function recallMessageById(id: number) {
-  fmsMessages.recallId(id);
+  fmsMessages?.recallId(id);
```

Comment:
This thing could probably be reworked to be a class with the bus etc. passed in as constructor properties so that we can initialize things in their property declaration and not need optional accesses like this.